### DISCUSSION TITLE
Welcome to StateSmith Discussions!

### DISCUSSION THREAD

**Discussion by adamfk** [StateSmith/StateSmith#24] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

I'm new to this, but I wanted a discussion space and github has one so let's check it out.

**Comment by mmcmahon98** [comment]
Hello All.
Our company currently has been developing using the Stateless nuget syntax.  I love how easy it is to use StateSmith with draw.io to create states.  It was said in one of the videos that you can use a Stateless code generator with StateSmith.  Has this every been developed?  Thanks to all the contributors here.  Looking forward to the response to my question.

Regards,
Mike

**Reply by adamfk** [reply]
Hi Mike, I don't believe anyone has started work on having StateSmith output Stateless code, but I'd be happy to help you with it if you are interested :)
